libraries/galera.rb

```diff
@@ -0,0 +1,55 @@
+module Galera
+  def self.cluster_running?(hosts, port)
+    commands = hosts.map { |host| spawn("echo '' | nc -w 10 #{host} #{port}", :out => "/dev/null") }
+    all_status = commands.map { |pid| Process.wait2(pid)[1].exitstatus  }
+    return all_status.min == 0
```

Comment:
OPT: linting

replace `return all_status.min == 0` by `all_status.min.zero?`
(return is useless at the end of the method)